src/MultiMessageReceiver.sol

```diff
@@ -186,10 +199,37 @@ contract MultiMessageReceiver is IMultiMessageReceiver, ExecutorAware, Initializ
     function _updateReceiverAdapter(address _receiverAdapter, bool _add) private {
         if (_add) {
             _addTrustedExecutor(_receiverAdapter);
+
+            _checkDuplicates(trustedExecutor);
```

Comment:
`_addTrustedExecutor()` would already ensure there are no duplicates in `trustedExecutor` right? is there a reason we need this separate `_checkDuplicates()`